README.md

```diff
@@ -10,6 +10,16 @@ npm install @inrupt/solid-client solid-auth-client
 
 See https://inrupt.github.io/solid-client-js/docs/
 
+# Related libraries
+
+## Authentication
+
+To send authenticated requests to your Pod and retrieve private data, you can use [solid-client-authn-browser](https://github.com/inrupt/solid-client-authn).
```

Comment:
I'd link to the package they'd be using, rather than to the source code (which, by the way, I don't see listed on the npm page yet - be sure to add the `repository` field to the package.json of solid-client-authn):

```suggestion
To send authenticated requests to your Pod and retrieve private data, you can use [solid-client-authn-browser](https://www.npmjs.com/package/@inrupt/solid-client-authn).
```